sound/soc/sof/topology.c

```diff
@@ -2633,19 +2633,35 @@ static void sof_dai_set_format(struct snd_soc_tplg_hw_config *hw_config,
 	}
 }
 
-/* set config for all DAI's with name matching the link name */
+/*
+ * Send IPC and set the same config for all DAIs with name matching the link
+ * name. Note that the function can only be used for the case that all DAIs
+ * have a common DAI config for now.
+ */
 static int sof_set_dai_config(struct snd_sof_dev *sdev, u32 size,
 			      struct snd_soc_dai_link *link,
 			      struct sof_ipc_dai_config *config)
 {
+	struct sof_ipc_reply reply;
```

Comment:
I still prefer keeping the scope as small as possible. In this case it would mean defining both `reply` and `ret` in the code block where they're really needed.

Reply:
> I still prefer keeping the scope as small as possible. In this case it would mean defining both `reply` and `ret` in the code block where they're really needed.

Thanks @lyakh Updated.